go.mod

```diff
@@ -109,8 +109,8 @@ require (
 	gopkg.in/inf.v0 v0.9.1 // indirect
 	gopkg.in/yaml.v2 v2.4.0 // indirect
 	gopkg.in/yaml.v3 v3.0.1 // indirect
-	k8s.io/apiextensions-apiserver v0.24.3 // indirect
-	k8s.io/code-generator v0.24.3 // indirect
+	k8s.io/apiextensions-apiserver v0.25.4 // indirect
+	k8s.io/code-generator v0.25.4 // indirect
```

Comment:
prefer to patch the indirect import via the `repalce` section.

Reply:
In the [vm-import-controller](https://github.com/harvester/vm-import-controller/pull/13/files#:~:text=k8s.io/code%2Dgenerator%20v0.25.4%20//%20indirect) this was the same approach used. Unless we have a version conflict or something, I don't think it's necessary to do a manual replace